src/olympia/addons/admin.py

```diff
@@ -358,25 +365,27 @@ def reviewer_links(self, obj):
         # provided by annotations made in get_queryset()
         if obj._listed_versions_exists:
             links.append(
-                '<a href="{}">{}</a>'.format(
+                (
                     urljoin(
                         settings.EXTERNAL_SITE_URL,
                         reverse('reviewers.review', args=['listed', obj.id]),
                     ),
-                    _('Reviewer Tools (listed)'),
+                    _('Review (listed)'),
```

Comment:
These changes are just to gain some horizontal space and be consistent, both in terms of copy and styling.